src/main/java/com/google/sps/servlets/HandlePhotoShoppingServlet.java

```diff
@@ -129,23 +140,25 @@ private String getQuery(String photoCategory, byte[] uploadedImageBytes)
         } catch (PhotoDetectionException exception) {
           throw exception;
         }
-        return productShoppingQuery;
+        queryResults.add(productShoppingQuery);
+        return queryResults;
       case "shopping-list":
         TextDetectionAPIImpl textDetectionAPI = new TextDetectionAPIImpl();
         ImageTextDectector imageTextDectector = new ImageTextDectector(textDetectionAPI);
 
-        String shoppingQuery;
+        List<String> shoppingQueries = new ArrayList<>();
         try {
-          shoppingQuery = imageTextDectector.extractShoppingList(uploadedImageBytes);
+          shoppingQueries = imageTextDectector.extractShoppingList(uploadedImageBytes);
```

Comment:
How about using {@code queryResults} here as well, to avoid creating a new list?

Reply:
Will do that. Thanks for suggesting :)